model/GamutCompress.blink

```diff
@@ -28,163 +25,28 @@ kernel GamutCompression : public ImageComputationKernel<ePixelWise> {
 
```

Comment:
Github is lame here, but we could certainly trim `float pi;` and `pi = 3.14159265359;` as they are not used anymore.